src/components/PartnerLogos.js

```diff
@@ -2,7 +2,10 @@ import { Grid, Typography } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
 import React from "react";
 
+import car from "@/sensorsafrica/assets/images/partners/car.png";
+import godown from "@/sensorsafrica/assets/images/partners/godown.png";
```

Comment:
Can we ask for a version with white background and black text? If not, try inverting it in any photo editing software.